test/vault/Vault.settleBadDebt.test.ts

```diff
@@ -90,7 +92,6 @@ describe("Vault settleBadDebt (assume zero IF fee)", () => {
         await addOrder(fixture, bob, 500, 1000000, 0, 150000)
 
         // Carol will liquidate Alice's position
-        await clearingHouseConfig.setBackstopLiquidityProvider(carol.address, true)
```

Comment:
Since we already release new liquidation mechanism, the bad debt liquidation can be done by anyone not just configured BLPs. And BackstopLiquidityProvider will be deprecated at this release, check [here](https://github.com/perpetual-protocol/perp-curie-contract/pull/30/files#diff-19ab1b74485b7ebdbed9a7cbec8e79ee43e547847490d52650e06ae8c72f3729L88).